scalaparse/shared/src/test/scala/scalaparse/unit/FailureTests.scala

```diff
@@ -123,7 +123,7 @@ object FailureTests extends TestSuite{
         |  1
         |}
       """.stripMargin,
-      expected = """ ";" | Newline.rep(1) | End """,
+      expected = """ ";" | Newline.rep(1) | End() """,
```

Comment:
Can we give `End` and all the other no-arg parsers `toString` methods without the `()`, for backwards-compatiblity for now?